docs/setup/installation.md

```diff
@@ -2,50 +2,87 @@
 title: Installation
 ---
 
-SourceCred currently support running from a Node environment or using Docker.
-_In a future release we're looking to add easier options._
+The easiest way to install SourceCred is to fork the [example repo](https://github.com/sourcecred/example-instance). This repo contains scripts that will install SourceCred locally and set up an example instance of SourceCred. This instance can then be configured to include just the plugins you want and point to the data sources you care about.
 
-## Node environment
+If you want to build SourceCred from source (for instance to get bleeding edge features) or to develop on it, see the [Development Setup instructions](#development-setup).
 
-The Node environment is useful for development and running from source code.
```

Comment:
Let's unmention docker, it's not really supported yet.

Reply:
Done